sources/packages/api/src/auth/roles.enum.ts

```diff
@@ -2,5 +2,34 @@
  * Application roles to be used in alongside with RolesGuard.
  */
 export enum Role {
```

Comment:
Are these roles AEST specific? If so we can rename the enum to AESTRoles?

Reply:
As discussed, I believe this is a generic Role.

Reply:
May be the enum when it was created is to represent generic roles.. But all the aest roles in this are aest-specific roles.